### ISSUE TITLE
Recipe/item changes in 2.4.2r1

### ISSUE THREAD

**Issue by elementography** [Beyond-Reality/BeyondRealityModPack#438]
I've noticed the following in the above release.  I confirmed by rolling back a version with the same save and all items returned.

Recipe for Heavy Duty Alloy Ingot (used to make T1 Heavy Duty Plate) is no longer available:

![capturec](https://cloud.githubusercontent.com/assets/8016868/19477073/181ea12c-956f-11e6-9ef1-1441cf7f475c.PNG)

The following items are no longer valid (not sure if intentional):

![capturea](https://cloud.githubusercontent.com/assets/8016868/19477094/2c3579ba-956f-11e6-9c25-b31693ee6eab.PNG)

GT Dark Steel dust
GT False Ruby Dust
GT False Ruby
GT Lanthanum Dust
GT Small false Ruby

They change to a no icon item with an ID like gt.metaitem.01.2512.name 5885:2512 etc.

Looks like the recipe for blizz powder or similar was changed as well as I have AE recipe for this which appears the same as the above items.

Let me know if you need more deets.

**Comment by leagris** [comment]
@elementography The Heavy Duty Ingot recipe is hotfixed if you download this replacement script https://raw.githubusercontent.com/Beyond-Reality/BeyondRealityModPack/master/BR1710/scripts/galacticraft.zs

The other item not showing in NEI is just intentional optimization from the new Gregtech material system.

**Comment by elementography** [comment]
@leagris thanks for that, it has addressed the issue with the Heavy Duty Ingot.  

With the items I mentioned, I presumed that it was to do with the optimisation.  They aren't showing in NEI, but they are still available in the world as shown below:

![captureg](https://cloud.githubusercontent.com/assets/8016868/19515478/457dfc56-962a-11e6-97a7-138568d42dd6.PNG)

![capturek](https://cloud.githubusercontent.com/assets/8016868/19515480/49b3623e-962a-11e6-84c5-49b2a907a88c.PNG)

Not a major issue as I can always trash them if they're no longer used, but, just figured I'd bring it up.

I noticed after my initial post that two of the three Rocket Engines from GT++ can no longer be crafted (verified again by checking in the previous version).  Both the Basic Rocket Engine (1159:753) and Turbo (1159:755) have no recipe now, but did prior to 2.4.2r1.  The Advanced can be crafted.

![capturz](https://cloud.githubusercontent.com/assets/8016868/19515690/0e897710-962b-11e6-989c-9e06a14c46d9.PNG)
